docs/user-guide/manual-install.md

```diff
@@ -0,0 +1,186 @@
+
+1. We'll start by deploying the `quote` service. Save the below configuration into a file named `quote.yaml`. This is a basic configuration that tells Kubernetes to deploy the `quote` container and create a Kubernetes `service` that points to the `quote` container.
+
+   ```
+   ---
+   apiVersion: v1
+   kind: Service
+   metadata:
+     name: quote
+     namespace: ambassador
+   spec:
+     ports:
+     - name: http
+       port: 80
+       targetPort: 8080
+     selector:
+       app: quote
+   ---
+   apiVersion: apps/v1
+   kind: Deployment
+   metadata:
+     name: quote
+     namespace: ambassador
+   spec:
+     replicas: 1
+     selector:
+       matchLabels:
+         app: quote
+     strategy:
+       type: RollingUpdate
+     template:
+       metadata:
+         labels:
+           app: quote
+       spec:
+         containers:
+         - name: backend
+           image: quay.io/datawire/quote:0.2.7
+           ports:
+           - name: http
+             containerPort: 8080
+   ```
+
+2. Deploy the `quote` service to the cluster by typing the command `kubectl apply -f quote.yaml`.
+
+3. Now, create a `Mapping` configuration that tells Ambassador to route all traffic from `/backend/` to the `quote` service. Copy the YAML and save to a file called `quote-backend.yaml`.
+
+   ```
+   ---
+   apiVersion: getambassador.io/v2
+   kind: Mapping
+   metadata:
+     name: quote-backend
+     namespace: ambassador
+   spec:
+     prefix: /backend/
+     service: quote
+   ```
+
+4. Apply the configuration to the cluster by typing the command `kubectl apply -f quote-backend.yaml`.
+
+5. Test the configuration by typing `curl -Lk https://<hostname>/backend/` or `curl -Lk https://<IP address>/backend/`. You should see something similar to the following:
+
+   ```
+   (⎈ |rdl-1:default)$ curl -Lk https://aes.ri.k36.net/backend/
+   {
+    "server": "idle-cranberry-8tbb6iks",
+    "quote": "Non-locality is the driver of truth. By summoning, we vibrate.",
+    "time": "2019-12-11T20:10:16.525471212Z"
+   }
+   ```
+
+## A single source of configuration
+
+1. In the Ambassador Edge Stack, Kubernetes serves as the single source of configuration. Changes made on the command line (via `kubectl`) are reflected in the UI, and vice versa. This enables a consistent configuration workflow. You can see this in action by navigating to the Mappings tab. You'll see an entry for the `quote-backend` Mapping that was just created on the command line.
+
+2. If you configured TLS, you can type `kubectl get hosts` to see the `Host` resource that was created:
+
+   ```
+   (⎈ |rdl-1:default)$ kubectl get hosts
+   NAME               HOSTNAME           STATE   PHASE COMPLETED   PHASE PENDING   AGE
+   aes.ri.k36.net     aes.ri.k36.net     Ready                                    158m
+   ```
+
```

Comment:
There's a Hosts tab as well as a Mappings tab. Similarly, `kubectl get mappings` works just like `kubectl get hosts`. Should we mention both options for both resource types? 

I understand this is copy-pasted from elsewhere, so perhaps my suggestion is best saved for a different PR.